integration-tests/vault-app/pom.xml

```diff
@@ -75,7 +75,6 @@
             <plugin>
                 <groupId>io.quarkus</groupId>
                 <artifactId>quarkus-maven-plugin</artifactId>
-                <version>${project.version}</version>
```

Comment:
isn't this reverting what you just blessed in https://github.com/quarkusio/quarkus/pull/4843 ?

Reply:
The issue is only if the plugin is in a profile. If it is not in a profile, it's OK. That's why I haven't remove the version in the profile below.